packages/web-app-preview/src/App.vue

```diff
@@ -350,13 +439,32 @@ export default defineComponent({
       }
       this.activeIndex--
       this.updateLocalHistory()
+    },
+    imageShrink() {
+      this.currentImageZoom = Math.max(0.1, Math.round(this.currentImageZoom * 0.8 * 20) / 20)
```

Comment:
Please introduce a method similar to the following:

```js
calculateZoom(zoom: number, factor: number) {
      return Math.round(zoom * factor * 20) / 20
    }
```

and use it in all the places where you have the respective Math.round statement at the moment.